src/Shared/Microsoft.DotNet.Services.Utilities/Utilities.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.DotNet.Services.Utilities
+{
+    public static class Utilities
```

Comment:
Having the namespace and the class name both be "Utilities" might be confusing.  Since it's now in a "Utitilties" DLL, I'd probably call this one "GitHelpers" or something.